tests/e2e/br/utils/tls/manager.go

```diff
@@ -0,0 +1,456 @@
+// Copyright 2021 PingCAP, Inc.
```

Comment:
This file should be put in a common dir that can be imported by all the other packages instead of under `br` dir, right?
The same comment for some other files above.

Reply:
BTW, can the file `tests/e2e/tidbcluster/tls.go` be reused?

Reply:
All utils in br will be changed to global in the future. But now it keeps in local to avoid confusion.

Reply:
`tests/e2e/tidbcluster/tls.go` will be replaced by `utils/tls` in the future. Comparing with current one, `utils/tls` combines multiple calls about creating certs and exports only one simple func.